pstress/mysqld_options_ms_80.txt

```diff
@@ -2011,9 +2427,6 @@
 --myisam-sort-buffer-size=2047
```

Comment:
Why do we still see myisam variables in the output file. Please note, pstress generated only InnoDB tables, so we should exclude all myisam variables